src/components/withScrollFader/index.tsx

```diff
@@ -0,0 +1,173 @@
+import React, {useCallback} from 'react';
+import {
+  StyleSheet,
+  // eslint-disable-next-line no-unused-vars
```

Comment:
It's impossible to read the code with all the lint comments. 
First, there's a way to ignore a block of code with eslint. 
Second, because of TS, this lint error won't break the build for now, I know it's annoying to see the red marks but for now ignore them, these things will be fixed/handled all together once we complete the TS migration.

Reply:
Removed